release-notes/v1_66.md

```diff
@@ -23,6 +23,16 @@ We really appreciate people trying our new features as soon as they are ready, s
 
 If you find issues or have suggestions, you can enter them in the [VS Code repository](https://github.com/Microsoft/vscode/issues) on GitHub.
 
+### Jupyter
```

Comment:
@rchiodo @DonJayamanne @amunger @minsa110 This was a quick peek at the release notes I was considering adding. Honestly, I feel like they are a bit scanty, so leaving as a draft to discuss. Should we maybe skip contributing to the main release notes this month?

Rich mentioned the New File change, but that doesn't seem super relevant to end users.

Reply:
@IanMatthewHuff Just a reminder that if you do decide to contribute a section to the 1.66 release notes, please target the vscode-docs vnext branch.
And if you are going to add images or .gifs, you should enable Git LFS https://github.com/microsoft/vscode-docs/blob/main/README.md#cloning

Reply:
@gregvanl Thanks, had the wrong branch targeted. I'll leave this open just for our discussion (since I'm not sure if we'll be contributing) and I'll close this and retarget for the final submit.